test/config/tests.js

```diff
@@ -71,15 +71,22 @@ exports['eslint-plugin-zapier'] = {
     assert.doesNotThrow(() => linter.executeOnText(''));
   },
 
-  'exposes a prettier config': () => {
-    const linter = makeLinter('prettier');
+  'exposes a node config': () => {
+    const linter = makeLinter('node');
     const config = linter.getConfigForFile('foo.js');
 
     assert.ok(Object.keys(config.rules).length > 0);
   },
 
-  'prettier config contains base config': () => {
-    const config = makeLinter('prettier').getConfigForFile('foo.js');
+  'exposes a browser config': () => {
+    const linter = makeLinter('browser');
+    const config = linter.getConfigForFile('foo.js');
+
+    assert.ok(Object.keys(config.rules).length > 0);
+  },
```

Comment:
could you an extra test that assert that the browser config sets the `browser` env to true?